docker/shared-build-env.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:8-stretch
```

Comment:
This was simply not working as the node:8 image is based on jessie rather than stretch. Not quite sure why this was working on TeamCity, but possibly it was in the docker cache on all the agents so hadn't been refreshed in a long time.